extensions/amp-story/1.0/progress-bar.js

```diff
@@ -193,6 +422,12 @@ export class ProgressBar {
 
     this.updateProgressByIndex_(segmentIndex, progress);
 
+    // If story was reloaded.
```

Comment:
Isn't this also called on initial render, even on the cover page?
Could we rename the `updateValuesForReload_` method so it better indicates what it does, like `getInitialSegmentIndex_`?

Reply:
Done.